packages/@storefront/sayt/src/previous-search-terms/index.html

```diff
@@ -0,0 +1,3 @@
+<gb-list items="{state.previousSearches}" item-alias="previous-search" item-props="{ onClick: props.onClick }">
```

Comment:
The value provided to `item-props` is missing the additional set of braces. Change to:

```
item-props="{ { onClick: props.onClick } }"
```